### PR TITLE
Use wrapper pillow to generate the images

### LINKED ISSUE #7: Use some wrapper instead of spawning a shell process
Use some wrapper or library to renderize the images, like Pillow.

### PR DESCRIPTION
**Pull request by luiz-meireles**
This pull request is to initially validate the new way of creating the images. 

Resolve #7

### CODE REVIEW

**Comment by luiz-meireles** [comment]
I forgot to add pillow dependence in the requirements

**Comment by luiz-meireles** [comment]
I'm working in some improvements related to the text generation.

**Comment by luiz-meireles** [comment]
I'm fixing layout problems.

**Comment by rodorgas** [comment]
Thanks for contributing! Removing the imagemagick dependency is amazing and requires a lot of code, thanks for the effort!

I'm having trouble running on my system, though. The keyword `embedded_color` [doesn't seem to exist](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html?highlight=multiline_text#PIL.ImageDraw.ImageDraw.multiline_text) on the latest stable version (7.2.0) of the Pillow's `ImageDraw.multiline_text` function. Maybe you're using an outdated version? I installed it from `requirements.txt`.

```
bot.py:158: TelegramDeprecationWarning: Old Handler API is deprecated - see https://git.io/fxJuV for details
  updater = Updater(TOKEN)
quote rodorgas
2020-10-12 21:17:23,228 - __main__ - WARNING - Update "..." caused error "multiline_text() got an unexpected keyword argument 'embedded_color'"
```

File: make_image.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+def make_image(photo: str, quote: str, name: str, result:  str, context: str = ""):
+
+    with Image.open(photo) as img:
+        quote = '“{}”'.format(quote)
+        img = img.convert("L")
+        img_caption = text_image(quote)
+        img_author = text_image(
+            name, font_size=int(FONT_SIZE * 0.5), padding=20)
+        img_context = text_image(
```
Comment (rodorgas) [inline make_image.py]: E quando for `/quote` sem contexto? Talvez precise de um condicional aqui.

Comment (rodorgas) [reply make_image.py]: Recebo o erro `caused error "'NoneType' object has no attribute 'expandtabs'` ao rodar um `/quote`, acredito que esteja relacionado.

**Review by rodorgas — CHANGES_REQUESTED** [review]

**Comment by luiz-meireles** [comment]
I will open a new clean pull request

**Pull request closed without merging**